server/server.ts

```diff
@@ -28,6 +29,14 @@ app.post(
   }
 );
 
+app.post('/createUser', authController.createUser, (req, res) => {
```

Comment:
how about we change the endpoint to '/signup'?

Reply:
updated!